addons/dragging/functions/fnc_handleScrollWheel.sqf

```diff
@@ -37,7 +37,9 @@ _position set [2, ((_position select 2) + _scrollAmount min (_maxHeight + 1.5))
 
 // move up/down object and reattach at current position
 detach _carriedItem;
-private _positionChange =  _position vectorDiff (getPosATL _carriedItem);
+
+// Uses this method of selecting position because setATL dosen't work on dedicated server
```

Comment:
`setPosATL` does work on remote objects, but from what I saw it didn't have immediate effects
At least in the same frame:
```
x setPosATL y;
(getPosATL x) != y
```